azure-cli: 2.0.18

derekbekoe opened the pull request:
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

ilovezfs commented on Formula/azure-cli.rb:
```diff
@@ -256,6 +266,11 @@ class AzureCli < Formula
     sha256 "d04bb2425086c3fe86f7bc48915290b13e798497839fbb18ab7f6dffcf98cc3a"
   end
 
+  resource "futures" do
```
did you intend to add this?

derekbekoe replied:
It's ignored lower below with `deps = resources.map(&:name).to_set - ["futures"]` and `pip install azure-cli homebrew-pypi-poet; poet -r azure-cli` includes it so I didn't remove it.

However, it would make sense to remove...

ilovezfs replied:
>It's ignored lower below with deps = resources.map(&:name).to_set - ["futures"]

Yep, that's why asked :)

>However, it would make sense to remove...

I'm happy either way.

derekbekoe commented:
@ilovezfs thanks for the quick remove!
I've removed the `futures` block. The formula was autogenerated and I didn't include a condition to move the `futures` block.
In next release, I'll remove it always. :)

Also, I removed the patch. :+1

ilovezfs commented:
You're welcome!

>Also, I removed the patch.

:bow: :heart: :tada: :+1:

>The formula was autogenerated 

Nice work. If only `ansible` were similarly automated … :see_no_evil:

>In next release, I'll remove it always.

Perfect!

derekbekoe commented:
I will remove the caveat in the next release or 2.
The formula is only now getting publicized.

ilovezfs commented:
Shipped! Thanks @derekbekoe! :rocket:

Note for future reference the standard bump subject is `azure-cli 2.0.18` not `azure-cli: 2.0.18` otherwise `brew pull` will bark at me

```
Warning: Subject should be: azure-cli 2.0.18
```

>I will remove the caveat in the next release or 2. The formula is only now getting publicized.

No worries.

derekbekoe commented:
> Note for future reference the standard bump subject is azure-cli 2.0.18 not azure-cli: 2.0.18 otherwise brew pull will bark at me

Noted.
